### PR TITLE
[OPT FAILURE] Fix gh-4131

### PR DESCRIPTION
**Pull request by nouiz**
Fix gh-4131

### CODE REVIEW

File: theano/sandbox/cuda/opt.py [hunk 1]
```diff
@@ -1023,11 +1023,11 @@ def local_gpu_flatten(node):
             return [gpu_flatten(host_input.owner.inputs[0], outdim)(
                 as_cuda_ndarray_variable(host_input.owner.inputs[0]))]
     if isinstance(node.op, tensor.Flatten):
-        x, shp= node.inputs
-        outdim = node.op.outdim
+        x = node.inputs
```
Comment (abergeron) [inline theano/sandbox/cuda/opt.py]: Perhaps you forgot a comma after the `x`to actually do unpacking?

**Comment by nouiz** [comment]
Fixed, added another small change in travis and rebased.

File: .travis.yml [hunk 2]
```diff
@@ -65,7 +65,7 @@ script:
   - cd $(python -c 'import theano; import os; print(os.path.split(theano.__file__)[0])')
   - echo "$PART"
   - cd -; cd Theano
-  - python -c 'import theano; print(theano.config)'
+  - python -c 'import theano; print theano.config.__str__(print_doc=False)'
```
Comment (abergeron) [inline .travis.yml]: This will break on python 3 because you removed the parentheses around the argument to `print`

**Comment by nouiz** [comment]
I fixed python 3.

**Pull request merged**